zebra-chain/src/amount.rs

```diff
@@ -619,6 +618,7 @@ mod test {
         let one = Amount::<C2>::try_from(1)?;
         let another_one = Amount::<C1>::try_from(1)?;
 
+        #[allow(clippy::eq_op)]
```

Comment:
I am so confused right now
```suggestion
        #[allow("clippy::eq-op")]
```